scripts/zones/Abyssea-Attohwa/npcs/Red_Ghost.lua

```diff
@@ -0,0 +1,24 @@
+--------------------------------------------
+    -- NPC: Red Ghost
+    -- Zone: Abyssea - Attohwa
+    -- !pos 23 -0.744 126 215
+--------------------------------------------
+
+local entity = {}
+
+entity.onTrade = function(player, npc, trade)
+end
+
+entity.onTrigger = function(player, npc)
+    player:startEvent(389)
+end
+
+entity.onEventUpdate = function(player, csid, option, npc)
+    
+end
+
+entity.onEventFinish = function(player, csid, option, npc)
+    player:setpos(16, -5.644, 139, 175)
```

Comment:
This bindings will be `nil`. 

```
    SOL_REGISTER("setPos", CLuaBaseEntity::setPos);
```

Reply:
I am not sure what that means? Need a capitol P on setPos?

Reply:
Yup, these are case sensitive.

Reply:
All done, thanks :)